ERCS/erc-N.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Associated Accounts
+description: A way to publicly associate two accounts with arbitrary contextual data
+authors: Steve Katzman (@stevieraykatz), Amie Corso (@amiecorso), Stephan Cilliers (@stephancill)
+discussions-to:
```

Comment:
Please create discussions topic in Eth Magicians with a link to this PR
https://ethereum-magicians.org/c/ercs/57